harvard-lts/CURIOSity
# Fixing Arabic link to homepage.

**Fixing Arabic link to homepage.**
* * *

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)

# What does this Pull Request do?
This fixes the link in the header to the homepage when the language is set to right-to-left languages like Arabic. When you are on a page with the language set, if you click on "CURIOSity Collections" in the header it will return you to the homepage in English with the body format aligned as left-to-right.

This is accomplished by doing two things:
* If the language is not English, `?locale=en` is added to the the "CURIOSity Collections"  header link.
* A meta tag is added to only the homepage that tells Turbolinks to not cache the page (https://github.com/turbolinks/turbolinks?tab=readme-ov-file#ensuring-specific-pages-trigger-a-full-reload)

# How should this be tested?

A description of what steps someone could take to:
* Rebuild CURIOSity with the `arabic-a11y` branch
* Go to an Exhibit that has the Arabic language available and switch to the Arabic language.
* Click on "CURIOSity Collections" in the header.
* Confirm that the homepage is in English and is aligned left-to-right.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz @f8f8ff
